### DISCUSSION TITLE
How often does streets gl synchronized with OSM?

### DISCUSSION THREAD

**Discussion by nchistov** [StrandedKitty/streets-gl#180] [Q&A] [answered]
I draw my village (add many houses, change roads etc.) But Streets GL doesn't show this.

**So, how often does streets gl synchronized with OSM?**

**Comment by kaffeeringe** [comment, marked as answer]
At the moment it doesn't get updated at all - for some months already. :-(

**Reply by pktiuk** [reply]
Could you tell us why?
